src/pages/home/page.jsx

```diff
@@ -2,16 +2,10 @@ import React from "react";
 import { getData } from "../../common/request";
 
 
-let HomePage = React.createClass({
-  statics: {
-    fetchData(params) {
-      return getData("/home");
-    }
-  },
-
+export default class HomePage extends React.Component {
   componentWillMount() {
     console.log("[HomePage] will mount with server response: ", this.props.data.home);
```

Comment:
Should these methods be arrows since they're using `this`?

(same as `componentWillMount: =>` in CoffeeScript)